pkg/grafana/folder-handler.go

```diff
@@ -151,6 +151,9 @@ func (h *FolderHandler) getRemoteFolder(uid string) (*grizzly.Resource, error) {
 		if err != nil {
 			return nil, err
 		}
+		if uid == "" {
```

Comment:
Should be the very first condition of the function IMO

Reply:
fair. Moved.